common/lib/protocol-definitions/api-report/protocol-definitions.api.md

```diff
@@ -345,7 +342,11 @@ export type ISequencedProposal = {
 
 // @public
 export interface IServerError {
-    errorMessage: string;
+    code: number;
+    // @deprecated
+    errorMessage?: string;
```

Comment:
Can we move the `errorMessage` property to the only extending interface: `ISummaryNack`, so at least we minimize the deprecated scope?